src/npm.rs

```diff
@@ -778,9 +778,11 @@ mod tests {
   fn range_parse() {
     // https://github.com/npm/node-semver/blob/4907647d169948a53156502867ed679268063a9f/test/fixtures/range-parse.js
     let fixtures = &[
+      // note: some of these tests previously used a `-0` pre-release (see link),
+      // but I removed them because it didn't seem relevant
       ("1.0.0 - 2.0.0", ">=1.0.0 <=2.0.0"),
-      ("1 - 2", ">=1.0.0 <3.0.0-0"),
```

Comment:
What happened here is we previously weren't parsing this, but I fixed it to parse it now... so it started erroring. This `-0` pre-release with `<` doesn't seem to have any effect with `npm:semver`, so I removed it from these tests.